dotnet/src/Connectors/Connectors.HuggingFace/Core/HuggingFaceClient.cs

```diff
@@ -136,14 +137,27 @@ internal HttpRequestMessage CreatePost(object requestData, Uri endpoint, string?
         string modelId = executionSettings?.ModelId ?? this.ModelId;
         var endpoint = this.GetTextGenerationEndpoint(modelId);
         var request = this.CreateTextRequest(prompt, executionSettings);
+
         using var httpRequestMessage = this.CreatePost(request, endpoint, this.ApiKey);
+        using var activity = ModelDiagnostics.StartCompletionActivity(endpoint, modelId, "HuggingFace", prompt, executionSettings);
```

Comment:
```suggestion
        using var activity = ModelDiagnostics.StartCompletionActivity(endpoint, modelId, "hugging_face", prompt, executionSettings);
```
?

Mind sending a PR to extend the list of constants to the OTel semconv (lmk if you need any help fighting with tooling)?

Reply:
I think we should use `huggingface` (without the `_`) since `OpenAI` is `openai`.